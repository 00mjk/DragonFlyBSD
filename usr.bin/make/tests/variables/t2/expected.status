$DragonFly: src/usr.bin/make/tests/variables/t2/expected.status,v 1.1 2005/03/02 10:58:58 okumoto Exp $
0
