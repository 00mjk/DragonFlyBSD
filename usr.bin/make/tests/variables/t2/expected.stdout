$DragonFly: src/usr.bin/make/tests/variables/t2/expected.stdout,v 1.1 2005/03/02 10:58:59 okumoto Exp $
Old: main.c globals.h  util.c util.h  map.c map.h  parser.y lexer.l  cmdman.1 format.5
Old: main.c globals.h  util.c util.h  map.c map.h  parser.y lexer.l  cmdman.1 format.5
New: main.c globals.h util.c util.h Map.c Map.h parser.y lexer.l cmdman.1 format.5
New: main_wrapper.c globals.h util.c util.h map.c map.h parser.y lexer.l cmdman.1 format.5
Integer Integer Integer Integer
Integer Variable Variable Variable
FOOFOOFOO BBB CCC
AAA BARBARBAR CCC
AAA BBB BAZBAZBAZ
AAA BBB CCC
AAA BBB CCC
AAA BBB CCC
AAA BBB CCC
AAA BBB CCC
AAA BBB CCC
